### PR TITLE
Add MetricsConfiguration for VaultServer

### PR DESCRIPTION
**Pull request by Shaad7**
Signed-off-by: Shaad7 <abdullah.alshaad@appscode.com>

### CODE REVIEW

File: charts/kubevault-metrics/templates/vaultserver/ops-kubevault-com-vaultopsrequest.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    kind: VaultOpsRequest
+  commonLabels:
+    - key: app
+      valuePath: .spec.vaultRef.name
+    - key: app_namespace
+      valuePath: .metadata.namespace
+    - key: type
+      valuePath: .spec.type
+  metrics:
+    - name: type
```
Comment (pkbhowmick) [inline charts/kubevault-metrics/templates/vaultserver/ops-kubevault-com-vaultopsrequest.yaml:17]: type is already added as common label, so another metrics named type seems redundant.

File: charts/kubevault-metrics/templates/vaultserver/ops-kubevault-com-vaultopsrequest.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    apiVersion: ops.kubevault.com/v1alpha1
+    kind: VaultOpsRequest
+  commonLabels:
+    - key: app
+      valuePath: .spec.vaultRef.name
+    - key: app_namespace
+      valuePath: .metadata.namespace
+    - key: type
+      valuePath: .spec.type
+  metrics:
```
Comment (pkbhowmick) [inline charts/kubevault-metrics/templates/vaultserver/ops-kubevault-com-vaultopsrequest.yaml:16]: Add some more metrics for VaultOpsRequest.
- created
- info (type can be added here as well)

**Review by pkbhowmick — CHANGES_REQUESTED** [review]

File: charts/kubevault-metrics/templates/vaultserver/ops-kubevault-com-vaultopsrequest.yaml (hunk 1 above)
Comment (Shaad7) [reply charts/kubevault-metrics/templates/vaultserver/ops-kubevault-com-vaultopsrequest.yaml:17]: Yes. Kinda redundent. This metrics is used only to check type of the ops request. We don't use the metrics in alerts or dashboard. We only use status metrics.

Comment (Shaad7) [reply charts/kubevault-metrics/templates/vaultserver/ops-kubevault-com-vaultopsrequest.yaml]: Removed

**Pull request merged**